chapters/develop-organize/projects/reading/README.md

```diff
@@ -60,8 +75,15 @@ A general structure for the project directory is:
 `-- checker.py
 ```
 
+RD: `checker.py` is an over simplification.
+You would have a checker / validator directory with checker / validation applications (Python, C etc.), reference input and output and others.
```

Comment:
It is intended to be that way. The methodology is supposed to be generic enough to be used in most cases. The idea of operating inputs and comparing them to expected outputs is common only in Computer Science.
IMO, other fields that admit checkers (Maths, Chemistry) will only compare the student's solution with a **single** set of expected values.